### PR TITLE
Adding Schema for Maps Block in Sidebar

### PR DESCRIPTION
**Pull request by iRohitSingh**
Adding Schema for Maps Block in Sidebar #2769

### CODE REVIEW

File: src/components/manage/Blocks/Maps/MapsData.jsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+  NoMaps: {
+    id: 'No map selected',
+    defaultMessage: 'No map selected',
+  },
+});
+
+const MapsData = (props) => {
+  const { data, block, onChangeBlock, schemaEnhancer } = props;
+  const intl = useIntl();
+  const schema = schemaEnhancer
```
Comment (tiberiuichim) [inline src/components/manage/Blocks/Maps/MapsData.jsx:23]: Do you have a use case where you have to pass down schemaEnhancer as a prop? The reason is that there's an automatic schemaEnhancer step done in the BlockDataForm component.

Comment (iRohitSingh) [reply src/components/manage/Blocks/Maps/MapsData.jsx:23]: ok! I haven't look into it. I will remove it if it is already done.

Comment (iRohitSingh) [reply src/components/manage/Blocks/Maps/MapsData.jsx]: removed it.

File: src/components/manage/Blocks/Maps/MapsData.jsx [hunk 2]
```diff
@@ -0,0 +1,51 @@
+import React from 'react';
```
Comment (tiberiuichim) [inline src/components/manage/Blocks/Maps/MapsData.jsx:1]: I don't think MapsData.jsx is needed, maybe just put this stuff in the MapsSidebar directly? In principle, we never do separate components for the sidebar in our blocks, just stash it all in the Edit component, but here it should be ok, as the edit has more stuff in it any way.

Comment (iRohitSingh) [reply src/components/manage/Blocks/Maps/MapsData.jsx]: I removed it. thank you

**Review by tiberiuichim — APPROVED** [review]

**Review by sneridagh — CHANGES_REQUESTED** [review]
@iRohitSingh we need the block to be extendable too, using"

```
import { withBlockExtensions } from '@plone/volto/helpers';

withBlockExtensions(...)
```

**Comment by nileshgulia1** [comment]
This PR should be brought up to date with Volto 15 standards.

File: src/components/manage/Blocks/Maps/schema.js [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+    {
+      id: 'default',
+      title: 'Default',
+      fields: ['url', 'align'],
+    },
+  ],
+
+  properties: {
+    url: {
+      title: props.intl.formatMessage(messages.MapsURL),
```
Comment (nileshgulia1) [inline src/components/manage/Blocks/Maps/schema.js:30]: We can use UrlWidget here.

**Review by nileshgulia1 — APPROVED** [review]

**Review by sneridagh — APPROVED** [review]

**Review by giuliaghisini — APPROVED** [review]

**Pull request merged**